plugins/reference-assets/mocks/action/action-navigation.json

```diff
@@ -26,6 +26,23 @@
             }
           }
         },
+        {
+          "asset": {
+            "id": "action-prev-without-icon",
+            "type": "action",
+            "value": "Prev",
+            "label": {
+              "asset": {
+                "id": "action-prev-id-without-icon",
+                "type": "text",
+                "value": "Go Back Without Icon"
+              }
+            },
+              "metaData": {
+                "role": false
```

Comment:
Is this to test that the transform will set it to `true`? 

If this is set explicitly, would we even want to change it?

Reply:
So, currently the user can specify the role property as true/false to show/hide the back icon. This part of story book shows a back button with a back icon

Reply:
if this is `false`, why would this show the icon?

Reply:
Hey, this frf is to show, the useage of `'role':'back'`. The one above this shows how to get the button without the back icon.